.github/workflows/ci.yml

```diff
@@ -85,6 +85,8 @@ jobs:
         run: make elixir-generate
       - name: run credo
         run: cd elixir && mix credo
+      - name: run dyalizer
+        run: cd elixir && mix dialyzer
```

Comment:
It's missing dependency cache and the `mix format --check-formatted` step.
I would copy the CI from wanda` and modify it, instead of writing the same here.

Reply:
the escritp install could be cached as well